agent/backend/pktvisor/scrape.go

```diff
@@ -168,7 +170,8 @@ func (p *pktvisorBackend) scrapeOpenTelemetry(ctx context.Context) {
 						return
 					}
 
-					p.receiver, err = p.createReceiver(ctx, p.exporter, p.logger)
+					metricsPath := strings.Replace(defaultMetricsPath, "__all", policyName, -1)
```

Comment:
We should build the full path, not replace __all here

Reply:
I changed it